blueocean-commons/src/main/java/io/jenkins/blueocean/commons/analytics/Analytics.java

```diff
@@ -71,7 +71,7 @@ public void track(TrackRequest req) {
     protected abstract void doTrack(String name, Map<String, Object> allProps);
 
     protected final String server() {
-        return InstanceIdentity.get().getPublic().getPublicExponent().toString(16);
+        return Hashing.md5().hashBytes(InstanceIdentity.get().getPublic().getEncoded()).toString();
```

Comment:
I believe SSH uses md5 hash of the public key as a fingerprint of the public key. So I am using that. We could use sha256 if you want?